README.md

```diff
@@ -60,9 +59,10 @@ Note that non-default networks require extra [firewall setup](https://cloud.goog
   Kubernetes clusters.
 * Volume resizing: CSI Filestore driver supports volume expansion for all supported Filestore tiers.
 * Topology preferences: For better performance, it is recommended to run
-  workloads in the same zone where the Cloud Filestore instance is provisioned in. In the
-  future, the location where to create a Cloud Filestore instance could be automatically
-  influenced by where the workload is scheduled.
+  workloads in the same zone where the Cloud Filestore instance is provisioned in. Zone preference
```

Comment:
+1 to moving this out of future features.

I'd also be a bit more explicit about how it works, with particular examples, so that it's clear what the order of preferences is. eg, (IIUC), specifying allowedTopologies=A,B and then binding with WaitForFirstConsumer=C will result in an instance in C; if binding is immediate then the topology would be A, etc.

Reply:
I have created a matrix of filestore create volume behavior based on the various knobs available. please take a look